modules/upgrade-cluster-version-parameters.adoc

```diff
@@ -0,0 +1,168 @@
+// Module included in the following assemblies:
+//
+// * upgrading/upgrading-aws.adoc
+
+[id='upgrade-cluster-version-parameters-{context}']
+= ClusterVersion paramaters
+
+The Cluster Version Operator (CVO) uses the following parameters to manage
+cluster versions and upgrades.
+
+.ClusterVersion parameters
+
+[cols="2,8a",options="header"]
+|===
+|Parameter |Description
+
+|`ClusterVersionSpec`
+|The specification that describes the cluster version state that you want. The
+CVO updates your cluster components to the specified versions.
+
+|`ClusterVersionSpec.clusterID`
+|Required. The unique RFC4122 UUID value that identifies your cluster.
+`xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx` hexadecimal value
+
+|`ClusterVersionSpec.desiredUpdate`
+|Optional. The cluster version that you want. If the current and intended
+version do not match, an upgrade starts. Set only a version from the list of
+recommended updates or your upgrade might fail.
+If an update exists with a version in the `availableUpdates` or history,
+you can specify the version field without specifying the image.
+
+If an upgrade fails, the Operator stops and reports the status of the failing
+component. Setting the `desiredUpdate` value to the previous version starts a
+rollback, but not all rollbacks succeed.
+
+|`ClusterVersionSpec.upstream`
+|Optional. Specify the preferred update server. If you do not specify a value,
+the CVO uses the best update server for the cluster and region.
+
+|`ClusterVersionSpec.channel`
+|Optional. Specify the channel to use to apply non-standard updates to the
+cluster.
+
+[IMPORTANT]
+====
+The default channel contains stable updates. Do not specify a
+`ClusterVersionSpec.channel` value on production clusters.
+====
+
+|`ClusterVersionSpec.overrides`
+|Optional. A list of overides for components that the CVO manages. Mark
```

Comment:
It will be useful to mention that this should be used only for debugging purpose and can prevent rollout of upgrades as it allows the cluster to drift to un-supported state.